module3/projects/index.md

```diff
@@ -3,3 +3,17 @@ layout: page
 title: Module 3 - Projects
 ---
 
+## Required Projects
+
+The following projects will be assigned and their completion is required to be promoted to module 2.  
```

Comment:
Typo -- `to be promoted to mod 4`

also once viewing party is merged (hopefully tomorrow!) this page will exist on main